changelogs/fragments/contains-filter.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- contains jinja2 filter - Add a contains jinja2 filter designed for use in map and selectattr tests (https://github.com/ansible/ansible/pull/45798)
```

Comment:
The first time I read this, I didn't realize that `contains` was the name of a new filter - I read it as a verb. I'd enclose the word `contains` in quotes or backticks to help the reader parse this entry more easily.

Reply:
Updated